CODE_OF_CONDUCT.md

```diff
@@ -37,9 +37,7 @@ However, sometimes these informal processes may be inadequate: they fail to
 work, there is urgency or risk to someone, nobody is intervening publicly and
 you don't feel comfortable speaking in public, etc. For these or other reasons,
 structured follow-up may be necessary and here we provide the means for that: we
-welcome reports by
-emailing [*Conduct-email*][conduct-mail] or
-in anonymous by filling out [*this form*][conduct-form].
+welcome reports by filling out [*this form*][conduct-form].
```

Comment:
Thinking it through, should this and the other bit about events actually link to the reporting section? That way folks can read the statement about privacy etc..?

Reply:
This is a good point. I've (hopefully) added links to the Reporting header in the 2 places you mentioned.

Reply:
Links work great, thanks!